### PR TITLE
fix: onnx package conflict during setup

### PR DESCRIPTION
**Pull request by ZiniuYu**
There is a package conflict with `onnxruntime` and `onnxruntime-gpu` in setup. If we install both packages, we are not able to run onnx using gpu. Installing only `onnxruntime-gpu` enables clip_server to run both on CPU and GPU.

This PR also corrects the instructions on running docker image in different runtimes.

This PR also replaces the broken images in test files

Notes: onnxruntime is fixed at 1.13.1 to pass the test_ranker
nvidia-tensorrt is fixed at 8.4.1.5 to pass trt related tests

### CODE REVIEW

File: server/setup.py [hunk 1]
```diff
@@ -51,9 +51,9 @@
     ],
     extras_require={
         'onnx': [
-            'onnxruntime',
             'onnx',
             'onnxmltools',
+            'onnxruntime',
```
Comment (numb3r3) [inline server/setup.py:56]: As you mentioned, we can always install `onnxruntime-gpu`?

**Review by numb3r3 — CHANGES_REQUESTED** [review]

File: server/setup.py (hunk 1 above)
Comment (ZiniuYu) [reply server/setup.py:56]: I've tested on x86 machines with/without GPU, both work fine. The CI failed for unknown reasons

File: server/setup.py [hunk 2]
```diff
@@ -51,12 +51,17 @@
     ],
     extras_require={
         'onnx': [
-            'onnxruntime',
+            'onnxruntime==1.13.1',
```
Comment (numb3r3) [inline server/setup.py:54]: I would prefer to use `onnxruntime<=1.13.1`

**Review by numb3r3 — CHANGES_REQUESTED** [review]

File: server/setup.py (hunk 2 above)
Comment (ZiniuYu) [reply server/setup.py:54]: let me do some permutations to pass the ci first 😅

**Review by numb3r3 — APPROVED** [review]
LGTM

**Pull request merged**